azurelinuxagent/agent.py

```diff
@@ -157,6 +162,8 @@ def main(args=[]):
                 agent.daemon()
             elif command == "run-exthandlers":
                 agent.run_exthandlers()
+            elif command == "show-configuration":
```

Comment:
>show-configuration [](start = 29, length = 18)

Does this just dump the waagent.conf, or the agent's interpretation of it?  Something else?